test/AlgebraicGeometry/ToricVarieties/total_space.jl

```diff
@@ -0,0 +1,47 @@
+@testset "Total space of direct sum of line bundles on toric space (set_attributes = $set_attributes)" for set_attributes in [true, false]
+
+	@testset "Test that some vb on P1 are Calabi-Yau" begin
+		P1 = projective_space(NormalToricVariety, 1; set_attributes)
+		for a in 0:5
+			for b in 0:5
+				la = toric_line_bundle(P1, [a])
+				lb = toric_line_bundle(P1, [-b])
+				X = total_space(la, lb)
+				@test is_smooth(X) == true
+				@test !is_fano(X)
+				@test !is_complete(X)
+				@test rank(picard_group(X)) == 1
+				@test dim(X) == 3
+				@test (degree(canonical_bundle(X)) == 0) == (a - b == -2)
+			end
+		end
+	end
+
+	@testset "Test that omega_S is Calabi-Yau for any S smooth surface" begin
+		@testset "S is Hirzebruch" begin
+			for a in 0:10
+				S = hirzebruch_surface(NormalToricVariety, a; set_attributes)
+				X = total_space(canonical_bundle(S))
+				@test is_smooth(X) == true
+				@test !is_fano(X)
+        @test !is_complete(X)
```

Comment:
Indention is weird here. I think you are mixing tabs and spaces. Please do not use tabs. In fact, please ideally read our [notes about code formatting in OSCAR](https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Code-formatting) 😄

Reply:
Thank you very much! I corrected the commit following your remark.

Reply:
It seemed that there was still a mixture of tabs and spaces. If the editorconfig is correctly acknowledged by VSCode, pressing tab should add two white space characters. (For me, they show as two centered dots, whereas a tab is shown as an arrow to the right).

I have taken the liberty to update your branch with a change, that turns all tabs into two whitespace characters. If I did not make a mistake, this should now agree with the requirements in https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Code-formatting.

Reply:
I am very sorry @mgemath , in my attempt to help I must have overwritten part (hopefully just a small part) of your desired changes. Really sorry. I just tried to fix my mistake, but could not. Maybe you immediately see what change I overwrote?

Reply:
Do not worry @HereAround. You were trying to correct my own mistakes. So it is me who must apologize in the first place. Anyway, it seems that you undone some of mine corrections. This is the list.

file: src/AlgebraicGeometry/ToricVarieties/Proj/constructors.jl
lines: 28, 32, 39, 43, 60, 65.

file: src/exports.jl
line: 1378.

file: test/AlgebraicGeometry/ToricVarieties/proj.jl
line: 43.

You can find the changes here: https://github.com/oscar-system/Oscar.jl/compare/8eb7eb4cfc9d1aca1f80cbbe76f129b5987e6d06..b68e44e915573330cea170b104528d7a5cb8c595

Reply:
Thank you. Just corrected those mistakes and added one more commit with a few minor optimizations. Let us see if the tests now pass.